src/general/index.ts

```diff
@@ -226,3 +226,54 @@ export function downloadFile(downloadConfig: { file: File | null; type: string;
     console.error('File download failed - ', err);
   }
 }
+
+
+/**
+ * Returns the value at given path from the source object. If path is not found then default value is returned.
+ *
+ * @param { { [key: string]: unknown } } obj - Source object
+ * @param {string} path - Path to desired key inside source object
+ *
+ * @remarks
+ * Provide a default value always to avoid unexpected behavior
```

Comment:
can be mentioned that this function has same behaviour as lodash getData method

Reply:
+1